tsconfig.module.json

```diff
@@ -3,13 +3,14 @@
   "extends": "./tsconfig.json",
   "compilerOptions": {
     "build": true,
-    "target": "es2020",
+    "target": "es2022",
     "module": "commonjs",
     "esModuleInterop": true,
     "declaration": true,
     "declarationMap": true,
     "noEmit": false,
     "emitDeclarationOnly": true,
+    "useDefineForClassFields": false,
```

Comment:
`useDefineForClassFields` defaults to `false` with `target: "es2020"` but `true` when `target: "es2022"`, causing different transpile behaviors. We can revisit what we want here in a separate PR targeting v9.